torch_audiomentations/utils/file.py

```diff
@@ -16,6 +17,11 @@ def find_audio_files(path):
     return files
 
 
-def load_audio(audio_file_path, sample_rate):
+def load_audio(audio_file_path, sample_rate=None, start=0, stop=None):
     """Loads the audio given the path of an audio file."""
-    return librosa.load(audio_file_path, sr=sample_rate)[0]
+    audio, source_sample_rate = soundfile.read(audio_file_path, start=start, stop=stop)
+
+    if sample_rate:
+        audio = resample_audio(audio, source_sample_rate, sample_rate)
+
+    return audio
```

Comment:
The sample rate should probably be returned. I guess we'll want it/ need it when it's `None` right?